neuror/cut_plane/detection.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, coefs: List[float],
             morphology: the morphology
             bin_width: the bin width
         '''
-        super(CutPlane, self).__init__(coefs[0], coefs[1], coefs[2], coefs[3], upward)
+        super().__init__(coefs[0], coefs[1], coefs[2], coefs[3], upward)
 
         if isinstance(morphology, Neuron):
```

Comment:
With this syntax it looks like morphology can be `None` or `[]` or `{}` but I don t think it makes sense.

I think you should add a : 
```
else:
     raise ...
```
Then you can remove the line 55.

Reply:
I have explicited the allowed types. The catch is that morphology=None is allowed, you may want to use a cut plane without its associated morphology.

Reply:
ok I see